python/ray/autoscaler/v2/instance_manager/ray_installer.py

```diff
@@ -80,6 +80,7 @@ def install_ray(self, instance: Instance, head_node_ip: str) -> bool:
             use_internal_ip=True,
             docker_config=docker_config,
             node_resources=self._config.get_node_resources(instance.instance_type),
+            node_labels=self._config.get_node_labels(instance.instance_type),
```

Comment:
So we are assuming labels are defined on per isntance type granularity?

I wonder if per-instance (maybe in the future) would make more sense?

Reply:
but i guess we don't have any per instance concept in autoscaler spec yet. Or maybe per instance type is good already?

Reply:
Yea. It's the same as resources, there is only per instance type granularity not per instance. Although you can create an node type for each instance.